crawl-ref/docs/options_guide.txt

```diff
@@ -69,7 +69,7 @@ The contents of this text are:
                 status_caption_colour, enemy_hp_colour, clear_messages,
                 show_more, small_more, show_newturn_mark, show_game_time,
                 equip_bar, animate_equip_bar, item_stack_summary_minimum,
-                mlist_min_height, mlist_allow_alternate_layout,
+                mlist_min_height, mlist_allow_alternative_layout,
```

Comment:
not entirely in favor of changing opts to satisfy the aus english thing, but you did at least provide backwards compatibility

Reply:
Crawl expects people to write option names down, and silently ignores mistakes. It seems to me like the place where spelling matters the most.

Compatibility is easier now than it was when this option was last renamed, so I had no reason not to.